packages/ERTP/src/issuerKit.js

```diff
@@ -11,10 +11,38 @@ import { makePaymentLedger } from './paymentLedger.js';
 import './types.js';
 
 /**
- * @type {MakeIssuerKit}
+ * @template {AssetKind} K
+ * The allegedName becomes part of the brand in asset descriptions. The
+ * allegedName doesn't have to be a string, but it will only be used for
+ * its value. The allegedName is useful for debugging and double-checking
+ * assumptions, but should not be trusted.
+ *
+ * The assetKind will be used to import a specific mathHelpers
+ * from the mathHelpers library. For example, natMathHelpers, the
+ * default, is used for basic fungible tokens.
+ *
+ *  `displayInfo` gives information to UI on how to display the amount.
```

Comment:
```suggestion
 *  `displayInfo` gives information to the UI on how to display the amount.
```

I realize you only moved these comments, but since I spotted typos, now seems the right time to fix them.